airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/generate_erd_schema/pipeline.py

```diff
@@ -0,0 +1,158 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+from __future__ import annotations
+
+import json
+import os
+import shutil
+import tempfile
+from dataclasses import dataclass
+from pathlib import Path
+from typing import TYPE_CHECKING, Any, List
+
+from dagger import Container
+
+from connector_ops.utils import ConnectorLanguage  # type: ignore
+from pipelines import main_logger
+from pipelines.airbyte_ci.connectors.consts import CONNECTOR_TEST_STEP_ID
+from pipelines.airbyte_ci.connectors.context import ConnectorContext, PipelineContext
+from pipelines.airbyte_ci.connectors.reports import Report
+from pipelines.airbyte_ci.connectors.test.context import ConnectorTestContext
+from pipelines.consts import LOCAL_BUILD_PLATFORM
+from pipelines.airbyte_ci.connectors.build_image.steps.python_connectors import BuildConnectorImages
+from pipelines.helpers.connectors.command import run_connector_steps
+from pipelines.helpers.connectors.format import format_prettier
+from pipelines.helpers.connectors.yaml import read_yaml, write_yaml
+from pipelines.helpers.execution.run_steps import STEP_TREE, StepToRun
+from pipelines.models.steps import Step, StepResult, StepStatus
+from pipelines.models.secrets import Secret
+
+if TYPE_CHECKING:
+    from anyio import Semaphore
+
+import google.generativeai as genai
+import dpath
+
+
+# TODO: how to pass secret in dagger?
```

Comment:
@artem1205 take a look at how `DOCKER_HUB_USERNAME` and password are passed around. I don't know how to do it, but I know we have a few examples like those.